src/server/controllers/getMemories.js

```diff
@@ -3,7 +3,7 @@ require('env2')('./config.env');
 const { getMemories } = require('./../../database/db_get');
 
 module.exports = (req, res) => {
-  getMemories(req.session.name, (error, memories) => {
+  getMemories('test', (error, memories) => {
```

Comment:
@des-des Yup - without we have to keep logging in to access the memories so we're going to change it back later. @joeylouise